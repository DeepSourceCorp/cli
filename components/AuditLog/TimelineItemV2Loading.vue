<template>
  <div class="space-y-4">
    <div class="h-3 w-full animate-pulse rounded-full bg-ink-200"></div>

    <div class="flex items-center gap-x-4">
      <div class="h-3 w-22 animate-pulse rounded-full bg-ink-200"></div>

      <div v-for="ii in 3" :key="ii" class="flex items-center justify-center gap-x-1">
        <div class="h-3 w-4 animate-pulse rounded-full bg-ink-200"></div>
        <div class="h-3 w-16 animate-pulse rounded-full bg-ink-200"></div>
      </div>
    </div>
  </div>
</template>

<script lang="ts">
import { Component, Vue } from 'nuxt-property-decorator'

@Component({ name: 'TimelineItemLoading' })
export default class TimelineItemLoading extends Vue {}
</script>
