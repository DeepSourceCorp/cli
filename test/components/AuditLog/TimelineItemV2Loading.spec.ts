import { render } from '@testing-library/vue'

import TimelineItemV2Loading from '~/components/AuditLog/TimelineItemV2Loading.vue'

describe('[[ TimelineItemV2Loading ]]', () => {
  it('renders the timeline item loading state', () => {
    const { html } = render(TimelineItemV2Loading)

    expect(html()).toMatchSnapshot()
  })
})
