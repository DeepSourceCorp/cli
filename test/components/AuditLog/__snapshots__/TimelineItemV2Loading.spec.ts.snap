// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[[ TimelineItemV2Loading ]] renders the timeline item loading state 1`] = `
"<div class=\\"space-y-4\\">
  <div class=\\"h-3 w-full animate-pulse rounded-full bg-ink-200\\"></div>
  <div class=\\"flex items-center gap-x-4\\">
    <div class=\\"h-3 w-22 animate-pulse rounded-full bg-ink-200\\"></div>
    <div class=\\"flex items-center justify-center gap-x-1\\">
      <div class=\\"h-3 w-4 animate-pulse rounded-full bg-ink-200\\"></div>
      <div class=\\"h-3 w-16 animate-pulse rounded-full bg-ink-200\\"></div>
    </div>
    <div class=\\"flex items-center justify-center gap-x-1\\">
      <div class=\\"h-3 w-4 animate-pulse rounded-full bg-ink-200\\"></div>
      <div class=\\"h-3 w-16 animate-pulse rounded-full bg-ink-200\\"></div>
    </div>
    <div class=\\"flex items-center justify-center gap-x-1\\">
      <div class=\\"h-3 w-4 animate-pulse rounded-full bg-ink-200\\"></div>
      <div class=\\"h-3 w-16 animate-pulse rounded-full bg-ink-200\\"></div>
    </div>
  </div>
</div>"
`;
